### ISSUE TITLE
Campaign Test Email Sends, Actual Campaign Emails are Caught in Spam

### ISSUE THREAD

**Issue by kmackinley** [gophish/gophish#1246]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 
0.5.0

Brief description of the issue: 
When creating a campaign, sending the test email on the campaign creation screen works without being caught in spam. When the campaign is created, its sending is delayed and then is caught in spam. I have tried different templates to no avail. What would change between the campaign test email or the actual campaign creation to make this happen? I can get the email to send without hitting spam on campaign creation if I remove the {{.URL}} tag, but that would not explain why it would work properly with the test email function at campaign creation screen. I've looked at previous similar issues about timezones and have already confirmed timezones are the same.

What are you expecting to see happen? :
If test email sends (on the campaign creation screen, not at the sending profiles test email function) with the proper settings and every working correctly, the campaign creation should send the same way.

What are you seeing happen?
Campaign creation emails are caught in spam, while test email from  very campaign creation screen works properly and is not blocked by spam.

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```

Please provide any terminal output that may be relevant below:

```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 -
